docs/security/customize-user-lockout.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Customize max login attempts and duration of user lockout
+description: Use Clerk to limit the number of times a user can attempt to log in before they are locked out of their account and customize how long such lockouts last.
+---
+
+# Customize max login attempts and duration of user lockout
+
+Clerk provides an Account Lockout feature in order to protect user credentials against brute force attacks. You can customize the number of times a login can be attempted before the account is locked,preventing further login attempts, and how long such a lockout lasts.
```

Comment:
```suggestion
Clerk provides an Account Lockout feature in order to protect user credentials against brute force attacks. You can customize the number of times a login can be attempted before the account is locked to prevent further login attempts, and how long such a lockout lasts.
```